hexrdgui/main_window.py

```diff
@@ -1730,3 +1735,55 @@ def action_apply_median_filter_toggled(self, b):
         # The dialog should have modified HexrdConfig's median filter options
         # already. Just apply it now.
         HexrdConfig().apply_median_filter_correction = b
+
+    def validateDragDropEvent(self, event):
+        mime_data = event.mimeData()
+        if not mime_data.hasUrls():
+            event.ignore()
+            return
+
+        # Make sure the selction(s) are files with a supported file extensions
+        paths = [url.toLocalFile() for url in mime_data.urls()]
+        supported_extensions = [
+            '.hexrd',
+            '.yml', '.yaml',
+            '.h5', '.hdf5',
+            '.tif', '.tiff',
+            '.png',
+            '.jpg', '.jpeg'
```

Comment:
So, for images, we actually support any type of image file that fabio supports (which is a lot of different kinds).  If a file comes in with an extension we don't recognize, maybe we should just try to open it as an image file and see what happens?

We also support frame cache formats in `.fch5` and `.npz` extensions. See [here](https://github.com/HEXRD/hexrdgui/blob/ff267ba3d0becf870d988b891bfadf7647558dec/hexrdgui/image_file_manager.py#L97-L100). We also support [hdf4 files too, right](https://github.com/HEXRD/hexrdgui/blob/ff267ba3d0becf870d988b891bfadf7647558dec/hexrdgui/image_file_manager.py#L64-L68)?

Reply:
Thanks! I actually thought I left a comment about this exact thing, must not have hit "comment" :upside_down_face: 

Makes sense though, my fear was that I was being too restrictive with the list which is in fact the case. I've dropped the list of expected file extensions and fall back to trying to open as an image.